### DISCUSSION TITLE
Add MacOS version

### DISCUSSION THREAD

**Discussion by IttyBitty997** [setvisible/DownZemAll#196] [Ideas]
Hello,

How to make in work in MacOS? I downloaded and installed the addon but the addon can't find the launcher.

I searched the download section but cannot find the launcher.

Thank you.

**Comment by kitpierce** [comment]
Also have this question - not averse to compiling it from scratch for MacOS (if there are some directions for the build...)

**Comment by setvisible** [comment]
The versions for Linux / MacOS X are preliminary versions, a lot of work remains, especially to build them correctly, and configure a proper build chain (on Travis maybe?).

I'm not an expert of MacOS X, so if someone knows how to do it, please feel free to modify the source code...

**Comment by setvisible** [comment]
See issue #2 

Haven't test on MacOSX but it works on Ubuntu + Firefox now (version >= 2.0.2). I wish we can close this ticket.
